runtime/runtime-core/api/runtime-core.api

```diff
@@ -1176,7 +1176,7 @@ public final class aws/smithy/kotlin/runtime/retries/policy/RetryErrorType : jav
 	public static final field ClientSide Laws/smithy/kotlin/runtime/retries/policy/RetryErrorType;
 	public static final field ServerSide Laws/smithy/kotlin/runtime/retries/policy/RetryErrorType;
 	public static final field Throttling Laws/smithy/kotlin/runtime/retries/policy/RetryErrorType;
-	public static final field Timeout Laws/smithy/kotlin/runtime/retries/policy/RetryErrorType;
+	public static final field Transient Laws/smithy/kotlin/runtime/retries/policy/RetryErrorType;
```

Comment:
Will this PR be shipped as a breaking change?

Reply:
Blerg yeah probably